Modules/System/Azure File Services API/src/AFSDirectoryContent.Table.al

```diff
@@ -0,0 +1,125 @@
+// ------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+// ------------------------------------------------------------------------------------------------
+
+/// <summary>
+/// Holds information about directory content in a storage account.
+/// </summary>
+table 8950 "AFS Directory Content"
+{
+    Access = Public;
+    InherentEntitlements = X;
+    InherentPermissions = X;
+    Extensible = false;
+    DataClassification = SystemMetadata;
+    TableType = Temporary;
+
+    fields
+    {
+        field(1; "Entry No."; Integer)
+        {
+            DataClassification = SystemMetadata;
+            Caption = 'Entry No.', Locked = true;
+            Access = Internal;
```

Comment:
Good question, not sure to be honest. As I mentioned this module was based on the module for blob storage, so it might be a leftover from there. I think the access modifier can be safely removed.